components/dropdown/dropdown-content-mixin.js

```diff
@@ -212,8 +212,11 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 	disconnectedCallback() {
 		super.disconnectedCallback();
 		window.removeEventListener('resize', this.__onResize);
-		document.body.removeEventListener('focus', this.__onAutoCloseFocus, true);
-		document.body.removeEventListener('click', this.__onAutoCloseClick, true);
+		if (document.body) {
```

Comment:
Wow.  We saw something similar, I think for more-less a little while ago.  Pretty crazy that we can't assume `document.body` will exist when this disconnects.  😬 There might be other places this could come up.